Documentation/ceph-toolbox.md

```diff
@@ -101,8 +109,37 @@ When you are done with the toolbox, you can remove the deployment:
 kubectl -n rook-ceph delete deployment rook-ceph-tools
 ```
 
-## Troubleshooting without the Toolbox
+## Toolbox Job
+
+If you want to run Ceph commands as a one-time operation and collect the results later from the
+logs, you can run a script as a Kubernetes Job. The toolbox job will run a script that is stored
+in a ConfigMap. The script has the full flexibility of a bash script.
+
+In this example, the `ceph status` command is executed when the job is created.
+
+```yaml
+kind: ConfigMap
```

Comment:
What is the point of making a ConfigMap script here instead of just putting this same thing in the Job manifest?

Reply:
The idea is to make it as simple as possible to run an arbitrary script with ceph commands. But that's a fair question since as a separate resource it likely just complicates things more. I'll look at adding it directly to the job spec like you did with the test wipe job.

Reply:
It would be cool if the script could just be an arg. I think that's possible.

Reply:
You mean like [this](https://github.com/rook/rook/blob/f792cd7c0472d8835030081a2cca88b1832ba7fa/tests/framework/installer/ceph_installer.go#L788)?

Reply:
yes!